client/settings/express-checkout/index.js

```diff
@@ -33,6 +35,28 @@ const ExpressCheckout = () => {
 		updateIsPlatformCheckoutEnabled,
 	] = usePlatformCheckoutEnabledSettings();
 
+	const [
+		enabledMethodIds,
+		updateEnabledMethodIds,
+	] = useEnabledPaymentMethodIds();
+
+	const updateStripeLinkCheckout = ( isEnabled ) => {
+		//this handles the link payment method checkbox. If it's enable we should add link to the rest of the
+		//enabled payment method.
+		// If false - we should remove link payment method from the enabled payment methods
+		if ( isEnabled ) {
+			updateEnabledMethodIds( [
+				...new Set( [ ...enabledMethodIds, 'link' ] ),
+			] );
+		} else {
+			updateEnabledMethodIds( [
+				...enabledMethodIds.filter( ( id ) => 'link' !== id ),
```

Comment:
```suggestion
				...enabledMethodIds.filter( id => 'link' !== id ),
```

Minor: Unnecessary `()`